build-monitor-plugin/src/main/webapp/scripts/controllers.js

```diff
@@ -24,6 +24,14 @@ angular.
                 return fetchJobViews().then(function (response) {
 
                     $scope.jobs = response.data.data;
+                    
+                    for (var i = 0; i < $scope.jobs.length; i++) {
+                        if( $scope.jobs[i].badges ) {
+                        	for (var j = 0; j < $scope.jobs[i].badges.length; j++) {
+                        		$scope.jobs[i].badges[j] = $sce.trustAsHtml($scope.jobs[i].badges[j]);
+                        	}
+                        }
+                    }
```

Comment:
Can we extract this into a function (like `sanitiseHTML` or something like that) and chain with `fetchJobViews` perhaps? 
`fetchJobViews().then(sanitiseHTML).then(function(response) {...`